bootstrap/bootstrap_reviews.py

```diff
@@ -0,0 +1,174 @@
+import csv
+import requests
+import datetime, json, os
+
+
+# A JWT generated for a service Moderator user
+AUTH = os.getenv("MODERATOR_AUTH")
+
+# The URL of the Moderator API
+API_URL = os.getenv("MODERATOR_API", "127.0.0.1:8080")
+
+# The number of reviews per category to import into Moderator
+REVIEWS_PER_CATEGORY = 100
+
+class moderator_client(object):
+
+    def __init__(self, auth, api_url):
+        self.api_url = api_url
+        self.headers = {
+            'content-type': "application/json",
+            'authorization': auth,
+            'cache-control': "no-cache",
+        }
+
+    def create_article(self, sourceId, title, text, url, category):
+        '''
+        Create an article in Moderator using the publisher API.
+        '''
+
+        data = {
+            "data": [
+                {
+                    "sourceId": sourceId,
+
+                    "categoryId": category,
+                    "title": title,
+                    "createdAt": str(datetime.datetime.now()),
+                    "text": text,
+                    "url": url,
+                }
+            ]
+        }
+
+        payload = json.dumps(data)
+
+
+
+
+        try:
+            print("Adding article", sourceId)
+
+            url = self.api_url + "/publisher/articles"
+            response = requests.post(url, data=payload, headers=self.headers)
+            status = response.status_code
+
+            if status != 200:
+                print("Received non-200 response from /publisher/articles:", status)
+
+            print("Response:", response.json())
+            print()
+
+        except Exception as e:
+            print("Error calling /publisher/articles:", e)
+
+    def create_comment(self, review_id, product_id, text, author_location, author_name):
+        '''
+        Create a comment in Moderator using the publisher API.
+        '''
+
+        data = {
+            "data": [
+                {
+                    "articleId": product_id,
```

Comment:
rename to use moderator terminology var names: s/product/article/ and s/review/comment/

Reply:
Ha, yes. Good catch. I originally wrote this to load in Amazon product reviews, hence the wacky names.